image_tools/src/burger.cpp

```diff
@@ -122,11 +135,19 @@ cv::Mat & Burger::render_burger(size_t width, size_t height)
     // bounce as needed
     if (x[b] < 0 || x[b] > width_i - burger_template.size().width - 1) {
       x_inc[b] *= -1;
-      x[b] += 2 * x_inc[b];
+      if (x[b] < 0) {
+        x[b] = 0;
+      } else {
+        x[b] = width_i - burger_template.size().width;
```

Comment:
Looks like `x_inc[b]` is unused?

Reply:
Correct it just truncates the movement at the border. It had already been moved in line 133.